docs/about/licensing.md

```diff
@@ -5,31 +5,38 @@ sidebar_label: Amplication’s licensing model
-- **Enterprise Edition** - add advanced operations, security, and commercially orientated features. The Enterprise edition is built on top of the Free edition and the additional functionality will be supplied as source-available. While users can view and download the code, ownership of the code remains with Amplication, meaning users cannot resell it. Use of the Amplication Enterprise edition is subject to the [Amplication Terms of Service](https://amplication.com/terms).
+- The Free Edition of Amplication includes our core functionality, based on Node.js. This edition is available under the [Apache 2.0 license](https://github.com/amplication/amplication/blob/master/LICENSE) and remains free for use on our managed service at [amplication.com](https://amplication.com).
+- Developers can freely access, modify, and distribute the source code available in this edition, fostering a collaborative and open development environment.
 
-## **Amplication’s Continuing Commitment to Open Source**[](https://docs.amplication.com/about/about/licensing/#amplications-continuing-commitment-to-open-source)
+### Enterprise Edition
 
-- Amplication remains fully committed to open source and promises to be a good steward of its open-source project, by ensuring that it continues to evolve with powerful new features. As we extend our Enterprise edition, we will ensure that this does not conflict with our open-source project.
-- When a feature has been contributed by members of the Amplication community, we won't move that feature to the Enterprise edition.
-- If a feature is planned for Free and the Enterprise edition, it will be released simultaneously in all. We won't introduce features into the Free Edition codebase with a fixed delay.
-- We will always make it clear what is proprietary and what is open-source code.
-- We will keep all features including the Enterprise Edition features for free on Amplication Cloud for the open-source community, individual developers, and small-scale projects.
+- The Enterprise Edition extends the Free Edition by incorporating advanced operational and security features. This edition introduces support for .NET applications, offering robust, scalable backend solutions that leverage popular .NET frameworks such as ASP.NET Core and Entity Framework Core.
+- This premium tier is designed for businesses seeking enhanced features like advanced data handling capabilities, increased automation, and comprehensive support for both Node.js and .NET applications.
+- While the Enterprise Edition's additional functionalities are source-available, meaning the source code can be viewed and downloaded, the ownership remains with Amplication. Consequently, this code is not open-source and cannot be resold. Usage is governed by the [Amplication Terms of Service](https://amplication.com/terms).
```

Comment:
It's more complicated....
The Enterprise Edition of node.js is partially open source (can be used) and partly source-available. The .NET is not source available.
So, I would avoid detailing all the options and avoid being apologetic. Something like- "The Enterprise Edition is partly open-source, partly source available, and partly Amplication proprietary and not source-available. The idea is to keep the Amplication business while doing the maximum for the community and our loved developers. "
I guess it's not perfect, and it can be improved. 
WDYT?